networkit/test/test_graph.py

```diff
@@ -90,6 +92,47 @@ def testRemoveMultiEdges(self):
 					self.assertFalse(G.hasEdge(u, v))
 				self.assertEqual(G.numberOfEdges(), 0)
 
+	def testGraphEventOperators(self):
+		up1 = nk.dynamics.GraphEvent(nk.dynamics.GraphEventType.NODE_ADDITION, 0, 0, 0)	
+		up2 = nk.dynamics.GraphEvent(nk.dynamics.GraphEventType.NODE_ADDITION, 1, 1, 1)
+		self.assertIsInstance(up1.__repr__(),str)
```

Comment:
I think that this is supposed to be used with `repr(up1)`. Also, we should test that the string representation works as intended, see the [implementation](https://github.com/networkit/networkit/blob/master/networkit/cpp/dynamics/GraphEvent.cpp#L17).